### ISSUE TITLE
Mehrfache Orte zu einer PLZ in Adressen

### ISSUE THREAD

**Issue by EwaldReinhardt** [SVWS-NRW/Schild3-BetaTest#1065]
Beim Ort im Katalog Allgemeine Adressen gibt es für die PLZ 58739 zwei Einträge:
![grafik](https://github.com/user-attachments/assets/267a0644-3d55-40bd-8d6c-b5cad1a9eb05)

In der Statistik wird nur einer erkannt (der mit Klammer).
Hier wäre eine Eindeutige Zuordnung gut

**Comment by JuergenRichter** [comment]
Nein, der Ortskatalog wird ja von der Schule verwaltet. Diese ist für die Einträge verantwortlich.

**Comment by AnneSchueller** [comment]
Dann schließe ich das Issue. In dem Fall Wickede muss die Schule dann selbst korrigieren.
